addons/website_sale_delivery/models/res_config_settings.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    is_installed_dhl = fields.Boolean()
```

Comment:
This is useless as `module_delivery_fedex` exists and is set. 

Same comment for the other fields.